### DISCUSSION TITLE
No NVIDIA driver was found for your system configuration

### DISCUSSION THREAD

**Discussion by MrkDrl** [ElPumpo/TinyNvidiaUpdateChecker#212] [Q&A] [answered]
Hi,

When checking for studio drivers, there is this prompt that pops up afterwards asking if I want to change to GRDs.

![Prompt](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/167992237/34844c43-daef-4454-831e-46a8a68877b2)

Is there any setting or command-line to disable this prompt?

**Comment by HawaiiBeach** [comment]
Hi. Well does your system accually support SD drivers, and is this an issue stemming from #172 ? Or does your system actually not support SD? What GPU?

**Reply by MrkDrl** [reply]
Yes, it supports studio drivers, I'm using a 4060 laptop. I get a similar error message to #172.
![Screenshot 2024-05-05 160606](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/167992237/20d5ef0b-6b7d-49af-82fb-22eca4a3e785)

**Comment by HawaiiBeach** [comment, marked as answer]
Hi. After a quick research I can confirm that the issue is about #172 which is about intermittent API issues.

Due to intermittent issues with the Nvidia API, there may be instances where the API returns incorrect driver data or doesn't return any data at all. These issues are not caused by our program but rather by inconsistencies within the Nvidia API itself.

The only solution is to wait a bit and try running TNUC again.

**Reply by MrkDrl** [reply]
Okay, but is it possible to disable the “Change driver type?” prompt that pops up afterwards?

**Reply by HawaiiBeach** [reply]
Well currently no.
